terminal/terminal.go

```diff
@@ -166,3 +182,68 @@ func getConfigFilePath(file string) (string, error) {
 	}
 	return path.Join(usr.HomeDir, configDir, file), nil
 }
+
+func loadConfig(cmds *Commands) *config {
+	fullConfigFile, err := getConfigFilePath(configFile)
+	if err != nil {
+		fmt.Printf("Unable to load config file: %v.", err)
+	}
+
+	f, err := os.Open(fullConfigFile)
+	if err != nil {
+		f, err = os.Create(fullConfigFile)
+		if err != nil {
+			fmt.Printf("Unable to create config file: %v.", err)
+		} else {
+			err = writeDefaultConfig(f, cmds)
+			if err != nil {
+				fmt.Printf("Unable to write default configuration: %v.", err)
+				_, err = f.Seek(0, os.SEEK_SET)
```

Comment:
Why seek on error? We should just return and let defer close the file descriptor for us.